### PR TITLE
chore: Add gitattributes for line endings

### PR DESCRIPTION
**Pull request by nschonni**
Ran `git add --renormalize .` after adding .gitattributes

### CODE REVIEW

**Comment by BenjaminArmstrong** [comment]
Can you explain the purpose of this change?  It is touching a bunch of sample code files.

**Comment by nschonni** [comment]
No problem! I think https://github.com/MicrosoftDocs/feedback/issues/362 covers why it's a good idea for git projects when you have contributors (or internal authors) working on different platforms. From that issue I think there might be some internal MS docs that explain it as well now, but I'm not sure where you folks keep that type of documentation.

**Comment by nschonni** [comment]
PS: if you add [?w=1 to file files tab url](https://github.com/MicrosoftDocs/Virtualization-Documentation/pull/1102/files?w=1), you can see the non-whitespace changes are the `.gitattributes` files

File: virtualization/community/.markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (nschonni) [inline virtualization/community/.markdownlint.json]: Oops, didn't mean to add this one, I'll rebase it out

**Comment by BenjaminArmstrong** [comment]
Thanks for the context!

**Pull request merged**